### PR TITLE
REV-58 ui-tweaks-phone-verification

### PR DESCRIPTION
**Pull request by IbrahimAldereni-socuimtech**
### Jira URL

https://optimumpartners.atlassian.net/jira/software/projects/REV/boards/46?selectedIssue=REV-58

### Description

UI enhancements on phone verification, 

### Dependencies in Other PRs,Repo

- figma link: https://www.figma.com/file/IKofGteYvhvMoXoafbBpgi/reverifi---Hover?node-id=0%3A2177

Screenshot from 2022-02-02 22-32-43

### CODE REVIEW

File: src/components/verify-phone/VerifyPhoneForm/index.jsx [hunk 1]
```diff
@@ -46,36 +47,58 @@ function VerifyPhoneForm() {
     if (target.nextSibling) {
       target.nextSibling.focus();
     }
+    if (target.name === 'd2') {
```
Comment (AhmadAlShwaiki) [inline src/components/verify-phone/VerifyPhoneForm/index.jsx:50]: Please explain to me what d2 is?

File: src/components/verify-phone/VerifyPhoneForm/index.jsx [hunk 2]
```diff
@@ -46,36 +47,58 @@ function VerifyPhoneForm() {
...
-              key={num}
-              {...register(`d${index}`)}
-              onInput={handleInput}
-            />
-          ))}
-        </InputsContainer>
-
+        <InputsWrapper>
+          <DigitInputsContainer>
+            {[0, 1, 2].map((num) => (
```
Comment (AhmadAlShwaiki) [inline src/components/verify-phone/VerifyPhoneForm/index.jsx:74]: Reasons behind division here?

File: src/components/verify-phone/VerifyPhoneForm/verify-phone-form.styles.js [hunk 3]
```diff
@@ -12,59 +12,63 @@ export const FormContainer = styled.form`
...
-    background-color: ${colors.gray};
-    color: ${colors.white};
-  }
 `;
 
 export const CodeContainer = styled.div`
   display: flex;
   flex-direction: column;
   gap: 2rem;
+  align-items: center;
```
Comment (AhmadAlShwaiki) [inline src/components/verify-phone/VerifyPhoneForm/verify-phone-form.styles.js:39]: background color should change to `alabaster` when field is filled

File: src/components/verify-phone/VerifyPhoneForm/index.jsx (hunk 1 above)
Comment (IbrahimAldereni-socuimtech) [reply src/components/verify-phone/VerifyPhoneForm/index.jsx:50]: i add comment above it, when 3rd input focused i jump to focus the 4th, cuz nextSibling won't work (they are seperated by div)

File: src/components/verify-phone/VerifyPhoneForm/index.jsx (hunk 2 above)
Comment (IbrahimAldereni-socuimtech) [reply src/components/verify-phone/VerifyPhoneForm/index.jsx:74]: to break them into 2 rows in mobile view

File: src/components/verify-phone/VerifyPhoneForm/index.jsx [hunk 4]
```diff
@@ -46,36 +47,59 @@ function VerifyPhoneForm() {
     if (target.nextSibling) {
       target.nextSibling.focus();
     }
+    // when 3rd input focused we focused the 4th (because  above logic (nexSibling won't work))
```
Comment (AhmadAlShwaiki) [inline src/components/verify-phone/VerifyPhoneForm/index.jsx:50]:
```suggestion
    // Allow continuous auto focus between selected digits. (digits are broken into two parts for mobile view) 
```

**Review by AhmadAlShwaiki — APPROVED** [review]

**Pull request merged**